tests/helpers.py

```diff
@@ -1,9 +1,9 @@
+import pytest
```

Comment:
The was actually right before:

https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
I fixed it because Flake8 was complaining.